hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/MembersUpdateOp.java

```diff
@@ -36,7 +36,7 @@
 import static java.util.Collections.emptyList;
 import static java.util.Collections.unmodifiableList;
 
-public class MembersUpdateOp extends VersionedClusterOperation {
+public class MembersUpdateOp extends AbstractClusterOperation {
```

Comment:
If I'm not thinking wrong, `MembersUpdateOp ` should remain `Versioned` for compatibility with 3.9.